aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +30,82 @@ const Icons = () => {
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
+            include different action items from the user, an icon can represent
+            different types of functionality.
+          </SubsectionText>
+        </Subsection>
+        <ButtonRow>
+          <Button
+            label="See in Figma"
+            href="https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons"
+            primary
+            target="_blank"
+            rel="noreferrer noopener"
+          />
+        </ButtonRow>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Developing with Grommet Icons" level={3}>
+          <SubsectionText>
+            For instructions on how to install and use grommet icons within your
+            application checkout the Grommet Icons site.
+          </SubsectionText>
+        </Subsection>
+        <Button
+          label="HPE Theme Grommet Icons"
+          href="https://icons.grommet.io/?theme=hpe"
+          primary
+          target="_blank"
+          rel="noreferrer noopener"
+        />
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Accessibility" level={3}>
+          <SubsectionText>
+            For screen reader accessibility there is a aria-label provided for
+            each icon.
+          </SubsectionText>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Default Icon" level={3}>
+          <Example
+            docs="https://icons.grommet.io/?theme=hpe"
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/foundation/icons/IconSimpleExample.js"
+            figma="https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons?node-id=261%3A0"
+          >
+            <IconSimpleExample />
+          </Example>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Icon Sizes" level={3}>
+          <SubsectionText>
+            The icon should be centered aligned with any text that is being used
+            next to it. The correct size icon should be paired with similar size
+            text.
+          </SubsectionText>
+          <Example code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/foundation/icons/IconSizeExample.js">
+            <IconSizeExample />
+          </Example>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Icon Color" level={3}>
+          <SubsectionText>
+            The icon default color is 'strong text' for the corresponding
+            light/dark mode For icons like Hpe logo the brand color will need to
```

Comment:
```suggestion
            light/dark mode. For emphasis or clarity, a color prop can
```

Reply:
Also, the Hpe logo, Aruba logo, Figma logo, and a few others are actually a weird case where the SVG has built in colors and for these, we actually want to use color="plain" to respect the intrinsic SVG color. I'm thinking this example could be changed to show a more generic component (maybe a 3 in a row) with different colors applied. Then, we create a different subsection for color="plain" examples.